frontend/src/lib/layout/Breadcrumbs.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+  import t from '$lib/i18n';
+  import type { ProjectPageData } from '$lib/page-data';
+
+  interface Breadcrumb {
+    name: string;
+    href: string;
+  }
+  let crumbs: Breadcrumb[] = [];
+
+  type CrumbMapperFunc = (token: string) => string;
+  type CrumbValue = string | CrumbConfig | CrumbMapperFunc | undefined | null;
+  type CrumbFallbackMapper = { _get: CrumbMapperFunc };
+  type CrumbWithChild = { _name: string | null };
+  interface CrumbConfig extends Record<string, CrumbValue>, Partial<CrumbFallbackMapper>, Partial<CrumbWithChild> {}
+
+  const CRUMB_CONFIG = {
```

Comment:
so if I understand correctly every page title needs to be declared here?

Reply:
Right. Well even every piece of a route for every page that includes Breadcrumbs.